libs/application/templates/party-application/src/lib/messages.ts

```diff
@@ -196,6 +196,33 @@ export const m: MessageDir = {
       defaultMessage: 'Sjá einungis meðmæli í vafa',
       description: 'Invalid signatures message',
     },
+    signatureInvalidTooltip: {
+      id: 'pa.application:endorsementList.tooltip',
+      defaultMessage: 'Meðmæli í vafa',
+      description: 'Invalid signature tooltip',
+    },
+  }),
+  bulkUpload: defineMessages({
+    uploadButton: {
+      id: 'pa.application:uploadButton.buttonTitle',
+      defaultMessage: 'Bæta við pappírsmeðmælum',
+      description: 'Button title',
+    },
+    fileFormatText: {
+      id: 'pa.application:uploadButton.fileFormatText',
+      defaultMessage: 'Tekið er við skjölum með endingu: .xlsx',
+      description: 'File format text',
+    },
+    uploadSuccess: {
+      id: 'pa.application:uploadButton.uploadSuccess',
+      defaultMessage: 'Pappírsmeðmælum hefur verið hlaðið upp!',
+      description: 'Upload succeeded text',
+    },
```

Comment:
these strings already exist under collectEndorsements, maybe move the "uploadFail" string to collectEndorsements section and delete this section one.. or vice versa, just avoid duplication 😛